recipes/dataframe/all/conanfile.py

```diff
@@ -65,7 +70,7 @@ def configure(self):
             "Visual Studio": "15",
             "gcc": "7",
             "clang": "6",
-            "apple-clang": "9.0",
+            # "apple-clang": "9.0",
```

Comment:
What you could so is For the old version leave 9 1 and for the new version set it to 10

Reply:
done